app/helpers/profiles_helper.rb

```diff
@@ -0,0 +1,16 @@
+module ProfilesHelper
+  def find_attribute_in(attr, address_type)
+    return address_type[attr] if address_type[attr].present?
```

Comment:
it can be written like this `address_type[attr].presence || default_placeholder(attr)`